src/advanced/node-management.md

```diff
@@ -2,8 +2,8 @@
 
 <!-- cspell:ignore nanos -->
 
-Exonum nodes can be controlled using RPC implemented via REST API. Managing
-endpoints are handled by Exonum core and are mainly purposed to receive
+Exonum nodes can be controlled using RPC implemented via REST API and WebSocket.
```

Comment:
I don't think that we can control exonum node via WebSocket. We can get some piece of information via WebSocket. And that all.

Reply:
updated